cmd/snap-bootstrap/cmd_scan_disk.go

```diff
@@ -176,7 +181,12 @@ func probeDisk(node string) ([]Partition, error) {
 			// For MBR we have to probe the filesystem for details
 			p, err := probeFilesystem(node, partition.GetStart()*ss64, partition.GetSize()*ss64)
 			if err != nil {
-				return ret, err
+				// On the pi, it has been observed during the installation of a preseeded image, that it
+				// can trigger udev, which retriggers snap-bootstrap scan-disk where in the non-gpt
+				// case we try to probe the filesystem too early, before it's formatted (so no LABEL).
+				// So log a warning, but continue processing other partitions.
+				logger.Noticef("WARNING: cannot probe filesystem on non-GPT partition: %s", err)
```

Comment:
Does logger works here?!? Very surprised. I think in theory we should write the `/dev/kmsg

Reply:
Haha i might have missed this - i'll find out in a moment if it works :-) in any case this should be printed elsewhere

Reply:
Just an FYI, we use `Logger.Noticef` as well on line 349

Reply:
I mixed up. I thought it was a generator. It is an udev program. Nevermind.